sdk/src/sdk.rs

```diff
@@ -84,6 +84,13 @@ pub fn set_config(key: &str, value: impl Into<serde_json::Value>) -> Result<(),
     )
 }
 
+pub fn delete_config<T: DeserializeOwned>(key: &str) -> Result<T, RequestError> {
+    Ok(serde_json::from_value(sync_request(
+        "config/delete",
+        KeyBody { key },
+  )?)?)
+}
```

Comment:
We should be able to make the code simpler by removing the type argument and returning a `Result<bool, RequestError>`. Then the command can just get a boolean value.

Reply:
Addressed, thank you!